### ISSUE TITLE
roadmap

### ISSUE THREAD

**Issue by philippedev101** [psibi/tldr-hs#5]
Is there any preference (priority) of what features are wanted? I saw that the node.js client has more command line options so that could be a thing .. https://github.com/tldr-pages/tldr-node-client#usage

**Comment by psibi** [comment]
@philippedev101 Please go ahead with anything you find interesting! For my day to day usage, the current option suffices. The `--list` and `--list-all` option seems interesting to me. But go ahead with whatever interests you (or any other interested contributor).

**Comment by psibi** [comment]
Note: You can use the file from assets: https://github.com/tldr-pages/tldr-pages.github.io/blob/master/assets/index.json to implement additional functionalities. Other client also seem to do the same.

**Comment by Ace-Cassidy** [comment]
I think that development of this client should be prioritized as it is the implementation currently in the ubuntu bionic packages. Ref: https://packages.ubuntu.com/search?keywords=tldr&searchon=names&suite=bionic&section=all

One thing I have noticed is there is some disparity between the usage listed when running ```tldr tldr``` Ref: https://github.com/tldr-pages/tldr/blob/master/pages/common/tldr.md vs actual usage for git commands e.g ```tldr git-branch``` works but ```tldr git branch``` does not, which seems counter to the documentation.

**Comment by psibi** [comment]
@Ace-Cassidy Are you using the latest version ? Seems to work for me:
```
~/g/tldr-hs (master) $ tldr --version
0.5.1
~/g/tldr-hs (master) $ tldr tldr
tldr
Simplified man pages.More information: https://tldr.sh.

 - Get typical usages of a command (hint: this is how you got here!):
   tldr {{command}}

 - Show the tar tldr page for linux:
   tldr -p {{linux}} {{tar}}

 - Get help for a git subcommand:
   tldr {{git checkout}}
~/g/tldr-hs (master) $ tldr git branch
git branch
Main git command for working with branches.More information: https://git-scm.com/docs/git-branch.

 - List local branches. The current branch is highlighted by *:
   git branch

 - List all branches (local and remote):
   git branch -a

 - Create new branch based on the current commit:
   git branch {{branch_name}}

 - Create new branch based on a specific commit:
   git branch {{branch_name}} {{commit_hash}}

 - Rename a branch (must not have it checked out to do this):
   git branch -m {{old_branch_name}} {{new_branch_name}}

 - Delete a local branch (must not have it checked out to do this):
   git branch -d {{branch_name}}
```

**Comment by wakeUPslow** [comment]
hello, I'm on Debian stable. I installed tldr via `sudo apt install tldr.` The apt package version is  0.6.4-1+b2 which is obviously behind the repository. If i clone the repository in order to get the latest version, do i need to apt remove or apt purge the older version prior too, or will it just update/integrate with the current version i have with out any trouble? Should I be using a different version or client? It don't seem like theirs a lot of activity here. I would like to contribute if possible, i think i speak for all the noobs when i say how helpful something like tldr is. Anyways thanks! konpai!

**Comment by psibi** [comment]
@wakeUPslow Yeah, I think purging it would be better and also removing the data files explicitly in case purging doesn't remove it automatically:

```
rm -rf ~/.local/share/tldr
```

Also, you don't need to compile the package as I provide static binaries that should work in any Linux distro: https://github.com/psibi/tldr-hs/releases/tag/v0.9.2
